src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2138,6 +2138,23 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task GetAsync_IdnHostName_SuccessStatusCodeInResponse()
+        {
+            HttpClientHandler handler = CreateHttpClientHandler();
+            using (var client = new HttpClient(handler))
+            {                
+                //international version of the Starbucks website
+                //punnycode: xn--oy2b35ckwhba574atvuzkc.com                
+                string idn = "\uc2a4\ud0c0\ubc85\uc2a4\ucf54\ub9ac\uc544.com";
```

Comment:
I would name this variable to something like "server".  "idn" implies that this string consists of punnycode character which it doesn't.  It is simply a hostname of a endpoint that just happens to have Unicode characters in the name.

And I would simplify this by bringing in the "http://" portion of the string here as well, i.e. 

```c#
string server = "http://\uc2a4\ud0c0\ubc85\uc2a4\ucf54\ub9ac\uc544.com";
```